core/vesting/vesting.go

```diff
@@ -343,7 +348,14 @@ func (e *Engine) broadcastSummary(ctx context.Context, seq uint64) {
 		PartiesVestingSummary: []*eventspb.PartyVestingSummary{},
 	}
 
-	for p, pRewards := range e.state {
+	parties := make([]string, 0, len(e.state))
+	for k := range e.state {
+		parties = append(parties, k)
+	}
+	sort.Strings(parties)
```

Comment:
Is this necessary? Things are sorted at the end of the loop?

Reply:
not necessary, no